src/checkbox.test.miscellaneous.ts

```diff
@@ -0,0 +1,105 @@
+import { html } from 'lit';
+import { styleMap } from 'lit/directives/style-map.js';
+import { expect, test } from './playwright/test.js';
+
+test('can be disabled', { tag: '@miscellaneous' }, async ({ mount, page }) => {
+  await mount(
+    html`<glide-core-checkbox label="Label" disabled></glide-core-checkbox>`,
+  );
+
+  const host = page.locator('glide-core-checkbox');
+
+  expect(host.waitFor).rejects.toThrow();
+});
+
+test(
+  'can be focused programmatically',
```

Comment:
[Most](https://github.com/CrowdStrike/glide-core/pull/1033/files#diff-2ef2f42233589fd4c3e46824202099dd677ec8b730c49ddd34eb162a1175c0ae) of our `*.test.focus.ts` suites test focus with form methods like `checkValidity()` and reportValidity()`. 

That why, I think, we no longer need `*.test.focus.ts` suites and can put to the few non-form focus tests in `*.test.miscellaneous.ts`